addons/website/views/snippets/s_quotes_carousel.xml

```diff
@@ -4,19 +4,13 @@
 <template id="s_quotes_carousel" name="Quotes">
     <section class="s_quotes_carousel_wrapper" data-vxml="001" data-vcss="001">
```

Comment:
Layout issue on mobile 🤔

| this PR | master |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/136f9120-aa6c-4199-92c0-049ac0b323ea) | ![image](https://github.com/user-attachments/assets/3d1d17db-2249-41f0-9922-99095b05f4b3) |